content/actions/use-cases-and-examples/building-and-testing/building-and-testing-rust.md

```diff
@@ -36,6 +39,37 @@ We recommend that you have a basic understanding of the Rust language. For more
 1. On the "Rust - by {% data variables.product.prodname_actions %}" workflow, click **Configure**.
 
    ![Screenshot of the "Choose a workflow" page. The "Configure" button on the "Rust" workflow is highlighted with an orange outline.](/assets/images/help/actions/starter-workflow-rust.png)
+
+{%- ifversion ghes %}
+   If you don't find the "Rust - by {% data variables.product.prodname_actions %}" workflow template, copy the following workflow code to a new file called `rust.yml` in the `.github/workflows` directory of your repository.
+
+   ```yaml copy
+   name: Rust
+
+   on:
+     push:
+       branches: [ "main" ]
+     pull_request:
+       branches: [ "main" ]
+
+   env:
+     CARGO_TERM_COLOR: always
```

Comment:
Does Actions filter out terminal escape codes now? The safest option is to set this to `never`
```suggestion
     CARGO_TERM_COLOR: never
```